### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by RealWillBrooks**

### CODE REVIEW

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,30 @@
...
+        this.catchPhrase = attributes.catchPhrase;
+    }
+}
+
+class Students extends Person {
+    constructor(attributes) {
+        super(attributes);
+        
+    }
+}
```
Comment (chrisipanaque) [inline assignments/lambda-classes.js]: MVP not complete.

File: assignments/prototype-refactor.js [hunk 2]
```diff
@@ -7,3 +7,100 @@ Prototype Refactor
...
+  console.log(mage.createdAt); // Today's date
+  console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
+  console.log(swordsman.healthPoints); // 15
+  console.log(mage.name); // Bruce
+  console.log(swordsman.team); // The Round Table
+  console.log(mage.weapons); // Staff of Shamalama
+  console.log(archer.language); // Elvish
+  console.log(archer.greet()); // Lilith offers a greeting in Elvish.
+  console.log(mage.takeDamage()); // Bruce took damage.
+console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
```
Comment (ghost) [inline assignments/prototype-refactor.js:106]: No console errors. Everything is correct. Excellent.

File: assignments/lambda-classes.js (hunk 1 above)
Comment (ghost) [inline assignments/lambda-classes.js:30]: `lambda-classes.js` file is not complete.

**Review by ghost — APPROVED** [review]
[x] Follows Lambda Git Flow
[] Frequent, clear, and descriptive commits
[] Code is sufficiently commented
[x] Code is organized and easy to read
[x] No errors in the console
[] MVP Complete
[] At least 1 stretch completed

Please remember to complete your MVP.

**Pull request merged**